src/lib/LUA/devLUA.cpp

```diff
@@ -241,29 +241,117 @@ static void luadevGeneratePowerOpts()
   *out = '\0';
 }
 
+#if defined(PLATFORM_ESP32)
+static void luahandWifiBle(struct luaPropertiesCommon *item, uint8_t arg)
```

Comment:
Maybe call this function `luahandConfirmationSendCmd` and also pass the 2 strings from the caller and a lambda for execute and cancel. 
We could do the same to the Simple version below and then these functions could move to lua.cpp as helpers.

Reply:
Adding a lambda wrapper adds 60 bytes of code to get the same functionality, so that's why I went this way with it. It is more generic sure but until we have more commands that do the same thing, there's no reason to use more code to do it. If the point is ever reached that the size of the setup is larger than the size of using lambdas, then it can be quickly refactored at that time.

The simple version can not use this function since it sends a lcsClick state, which would block things from working while connected (and those things require a connection too). There's code to check / change connectionState / reboot which none of those do, and also there's other variables being set / functions called. I think that it would end up the same size or larger to try to work these into just one function and becomes harder to follow because there's 2 separate behaviors that 7 different parameters would use. That's why I think 2 functions for 2 behaviors is warranted.

Reply:
Fair enough, I just thought it would be nice to have it generalised, because we'll be using it in the RX LUA too soon.